tools/http_benchmark.py

```diff
@@ -60,15 +60,32 @@ def hyper_http_benchmark(hyper_hello_exe):
 
 def http_benchmark(deno_exe, hyper_hello_exe, core_http_bench_exe):
     r = {}
+    m = {}
+
     # TODO Rename to "deno_tcp"
-    r["deno"] = deno_http_benchmark(deno_exe)
-    r["deno_net_http"] = deno_net_http_benchmark(deno_exe)
-    r["deno_core_single"] = deno_core_single(core_http_bench_exe)
-    r["deno_core_multi"] = deno_core_multi(core_http_bench_exe)
-    r["node"] = node_http_benchmark()
-    r["node_tcp"] = node_tcp_benchmark()
-    r["hyper"] = hyper_http_benchmark(hyper_hello_exe)
-    return r
+    denoHttp = deno_http_benchmark(deno_exe)
+    denoNetHttp = deno_net_http_benchmark(deno_exe)
+    denoCoreSingle = deno_core_single(core_http_bench_exe)
+    denoCoreMulti = deno_core_multi(core_http_bench_exe)
+    nodeHttp = node_http_benchmark()
+    nodeTcp = node_tcp_benchmark()
+    hyperHttp = hyper_http_benchmark(hyper_hello_exe)
+
+    dic = {
+        "deno": denoHttp,
+        "deno_net_http": denoNetHttp,
+        "deno_core_single": denoCoreSingle,
+        "deno_core_multi": denoCoreMulti,
+        "node": nodeHttp,
+        "node_tcp": nodeTcp,
+        "hyper": hyperHttp
+    }
+
+    for key, value in dic.items():
+        r[key] = value["req_per_sec"]
+        m[key] = value["max_latency"]
```

Comment:
It feels weird to munge the return types here rather than do:

```
new_data["req_per_sec"] = {k: v["req_per_sec"] for k, v in stats.items()}
new_data["max_latency"] = {k: v["max_latency"] for k, v in stats.items()}
```

in benchmark.py.